nodeup/pkg/model/kubeapiserver.go

```diff
@@ -160,6 +161,14 @@ func (b *KubeAPIServerBuilder) buildPod() (*v1.Pod, error) {
 		},
 	}
 
+	if os.Getenv("HTTP_PROXY") != "" || os.Getenv("HTTPS_PROXY") != "" {
+		container.Env = []v1.EnvVar{
+			{Name: "HTTP_PROXY", Value: os.Getenv("HTTP_PROXY")},
```

Comment:
In my experience `export HTTP_PROXY=` functions the same as `unset HTTP_PROXY`. There's an argument to be made for easy debugging (although maybe it goes both ways). Maybe it's neater with only the ones that get set. If anyone has a strong opinion, it's a quick change for me.